.snyk

```diff
@@ -2,6 +2,31 @@
 version: v1.22.1
 # ignores vulnerabilities until expiry date; change duration by modifying expiry date
 ignore:
+  SNYK-JS-XML2JS-5414874:
+    - '*':
+        reason: nodemailer-ses-transport needs to be upgraded to latest version in hof
+        expires: 2024-09-15T00:00:00.000Z
+        created: 2023-09-15T13:10:47.350Z
+  SNYK-JS-JSONWEBTOKEN-3180022:
+    - '*':
+        reason: notifications-node-client needs to be upgraded to latest version
+        expires: 2024-09-15T00:00:00.000Z
+        created: 2023-09-15T13:10:47.350Z
+  SNYK-JS-SEMVER-3247795:
+    - '*':
+        reason: notifications-node-client needs to be upgraded to latest version
+        expires: 2024-09-15T00:00:00.000Z
+        created: 2023-09-15T13:10:47.350Z
+  SNYK-JS-REQUEST-3361831:
```

Comment:
Hi, I'm a bit confused why there is gov notify and node-mailer? which one does rotm use?

Reply:
It uses GOV notify now which uses notifications-node-client under the hood :)
https://github.com/alphagov/notifications-node-client